azure-pipelines.yml

```diff
@@ -71,12 +71,12 @@ jobs:
           parameters:
             HelixSource: 'pr/dotnet/performance/$(Build.SourceBranch)' # sources must start with pr/, official/, prodcon/, or agent/
             HelixType: 'test/performance_$(_BuildConfig)/'
-            HelixTargetQueues: $(_HelixQueue) # Windows.Amd64.ClientRS4.DevEx.15.8.Perf # This will be whatever we call our perf pool
```

Comment:
Why are we removing these comments?

Reply:
Because _HelixQueue already specifies "what we want to call our perf pool." They were leftover from when I was initially writing the code.